addon/components/tag-post-list.hbs

```diff
@@ -0,0 +1,41 @@
+<div class="tag-post-list">
+  <h3
+    id={{this.titleId}}
+    class="text-lg tag-post-list-title"
+    data-test-tag-name
+  >
+    {{@tag.name}}
+  </h3>
+
+  {{get @tag.posts '1'}}
```

Comment:
what does this do? 🤔 is this something leftover from testing? 😂

Reply:
Oh, yes, that's from my experimentation :) Great catch! Sorry for my inattentiveness.

Reply:
ha ha don't worry about it! to be honest the PR was so good that I was tempted to not even raise this as an issue 😂

Reply:
Fixed!